flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliClientInputOutputSwitch.java

```diff
@@ -18,33 +18,14 @@
 
 package org.apache.flink.table.client.cli;
 
-import org.junit.rules.ExternalResource;
-
-import java.io.PrintStream;
-
 /**
  * Enables {@link org.apache.flink.table.client.SqlClient} to create a default terminal using {@link
  * System#in} and {@link System#out} as the input and output stream. This can allows tests to easily
  * mock input stream of the SqlClient by hijacking the standard stream.
  */
-public class TerminalStreamsResource extends ExternalResource {
-
-    public static final TerminalStreamsResource INSTANCE = new TerminalStreamsResource();
-    public static final PrintStream STD_OUT = System.out;
-
-    private TerminalStreamsResource() {
-        // singleton
-    }
-
-    @Override
-    protected void before() throws Throwable {
-        CliClient.useSystemInOutStream = true;
-    }
+public class CliClientInputOutputSwitch {
 
-    @Override
-    protected void after() {
-        CliClient.useSystemInOutStream = false;
-        // Reset the System.out
-        System.setOut(STD_OUT);
+    public static void useSystemInOutStream(boolean useOrNot) {
+        CliClient.useSystemInOutStream = useOrNot;
```

Comment:
I think it unnecessary to have a class for such a simple code. We can just place the code in `SqlClientTest` and add comments on it. Would better add `TODO: should be refactored in FLINK-22080` on it too.